roles/prepare-rancher-env/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+# defaults file for prepare-rancher-env
+rancher_image: rancher/rancher
+selenium_chrome_image: selenium/standalone-chrome
+
+packages:
+  - pytest
+  - selenium
+  - docker
+
+rancher_prefix: rancher-
+
+rancher_password: '{{ rancher_credentials.password }}'
+rancher_driver_location: https://github.com/opentelekomcloud/kontainer-engine-driver-otc/releases/download/v0.2.1/kontainer-engine-driver-otc-0.2.1-linux-amd64.tgz
```

Comment:
let's make this a variable coming from job (default points here, but job should be able to override it)